prime-router/metadata/organizations-local.yml

```diff
@@ -91,6 +91,16 @@
         transforms: { deidentify: false }
         format: HL7
 
+  - name: fl-phd
+    description: Florida Department of Health
+    services:
+      - name: elr
+        topic: covid-19
+        schema: fl/fl-covid-19
+        jurisdictionalFilter: [ "matches(ordering_facility_state, FL)" ]
```

Comment:
Check with the PHD on what rule they want for the long term.   In AZ, it was if the resident's address is in AZ __or__ if the ordering facility was in AZ.